README.md

```diff
@@ -91,6 +97,28 @@ Eventually [all `std` collection types will be parameterized by an
 allocator](https://github.com/rust-lang/rust/issues/42774) and we can remove
 this `collections` module and use the `std` versions.
 
+### Boxed
+
+When the `"boxed"` cargo feature is enabled, a fork of `std::boxed::Box`
+library is available in the `boxed` module. This `Box` type is modified to allocate
+its space inside `bumpalo::Bump` arenas.
+
+```rust
+use bumpalo::{Bump, boxed::Box};
```

Comment:
Is this the doc example you need the feature for?

We can do something like https://github.com/rust-fuzz/arbitrary/blob/master/src/unstructured.rs#L45 and that way keep the doc tests passing on all channels.

Thanks for adding this example!

Reply:
No, no. This stuff
https://github.com/fitzgen/bumpalo/pull/76/files#diff-6616a4c448a262bac3046bd5e1ac68cbR91

Reply:
Is there a reason we can't do the same thing that I linked to in those locations, instead of those nightly-only attributes?

Reply:
Those different things entirely.
This unstable attribute adds nice badge to the types and functions so that doc reader immediately see what features should be enabled instead of searching this info in the description or, even worse, open code to see that type is behind a feature after compilation fails.

See this module for example.
https://docs.rs/tokio/0.2.21/tokio/fs/index.html

Reply:
Ah, I see. I think it is fine to leave that information to normal comments, until the attributes are available on all channels.

Reply:
Ok. Then I remove those attributes.